index.html

```diff
@@ -0,0 +1,506 @@
+        <div class="px-4 border-x">
+          <div class="flex flex-col gap-5">
+            <h4 class="text-primary_blue text-lg font-bold">Why stripe</h4>
+            <h3 class="font-bold text-4xl md:w-[740px]">A technology-first approach to payments and finance</h3>
+          </div>
+          <div class="grid grid-cols-1 mt-8 gap-9 md:grid-cols-4 ">
+            <div class="flex flex-col gap-3">
+              <img src="./assets/images/tools.png" class="w-12" alt="tools image">
+              <h4 class="font-semibold">Close to the metal</h4>
+              <p>From <span class="text-primary_blue font-semibold hover:text-black cursor-pointer">direct
+                  integrations</span> with card networks and
+                banks to
+                checkout flows in the browser, we
+                operate on
+                and optimize at
+                every level of the financial stack.</p>
+            </div>
+            <div class="flex flex-col gap-3">
+              <img src="./assets/images/play.png" class="w-12" alt="tools image">
+              <h4 class="font-semibold"> Fastest-improving platform</h4>
+              <p> We release <span class="text-primary_blue font-semibold hover:text-black cursor-pointer">hundreds of
+                  features</span>
+
+                and improvements each year to help you stay ahead of industry shifts.
+                (On average, we deploy our production API 16x per day.)</p>
+            </div>
+            <div class="flex flex-col gap-3">
+              <img src="./assets/images/analytics.png" class="w-12" alt="tools image">
+              <h4 class="font-semibold">Battle-tested reliability</h4>
+              <p>Our systems operate standards with <span
+                  class="text-primary_blue font-semibold hover:text-black">99.99%+</span>
+                standards uptime and are highly scalable and redundant. Stripe is certified to
+                the highest compliance standards.</p>
+            </div>
+            <div class="flex flex-col gap-3 md:mb-36">
+              <img src="./assets/images/gear.png" class="w-12" alt="tools image">
+              <h4 class="font-semibold">Intelligent optimizations</h4>
+              <p>Our machine learning models train on <span
+                  class="text-primary_blue font-semibold hover:text-black cursor-pointer">billions</span>
+
+                billions of data points and help increase revenue across
+                conversion, fraud, revenue recovery, and more.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+
+    </section>
+
+    <div class="hidden lg:block mt-80">
+      <div class="bg-cyan-400 -mt-40 -skew-y-6 w-52 h-10 ml-10  "></div>
+      <div class="bg-primary_blue  -mt-6 opacity-80 -skew-y-6 w-80 h-10   "></div>
+    </div>
+    <!--Global scale-->
+    <section class=" relative -mt-[59px]">
+      <div class="-skew-y-6  w-full bg-primary_dark  h-[13vw] absolute top-0">
```

Comment:
Fix these conflicting sections, by adding some space between them on the mobile devices, compare your clone with the original one to get the clear image of how it should be:

**Yours:** 

![image](https://user-images.githubusercontent.com/58233753/203082586-a1741310-49b9-4432-9548-3d1f8c76fac1.png)

**Original**
![image](https://user-images.githubusercontent.com/58233753/203082689-f324888b-8be2-43cf-8dc7-98ace4334767.png)